### PR TITLE
TINY-4543: Added 5.1.5 changelog

### PR DESCRIPTION
**Pull request by lnewson**

### CODE REVIEW

File: changelog.md [hunk 1]
```diff
@@ -10,6 +10,12 @@ class: changelog
 
 {% capture changelog %}
 
+## Version 5.1.5 December 19, 2019
+* Fixed the UI not working with hybrid devices that accept both touch and mouse events.
+* Fixed bug with pasting image URLs when paste as text is enabled.
```
Comment (tylerkelly13) [inline changelog.md]: s/bug/a bug/ and s/paste as text/`paste as text`/

File: changelog.md [hunk 2]
```diff
@@ -10,6 +10,12 @@ class: changelog
...
+* Fixed the `charmap` dialog initially focusing the first tab instead of the search input.
```
Comment (tylerkelly13) [inline changelog.md]: focusing the s/first tab/first tab of the dialog/ instead of the search s/input/input field/   ?

File: changelog.md (hunk 1 above)
Comment (lnewson) [reply changelog.md]: Fixed

File: changelog.md (hunk 2 above)
Comment (lnewson) [reply changelog.md]: Fixed

**Pull request merged**